packages/fxa-payments-server/src/routes/Checkout/index.test.tsx

```diff
@@ -0,0 +1,141 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import noc from 'nock';
+
+function nock(it: any) {
+  //@ts-ignore
+  return noc(...arguments).defaultReplyHeaders({
+    'Access-Control-Allow-Origin': '*',
+  });
+}
+
+import {
+  PRODUCT_ID,
+  PRODUCT_REDIRECT_URLS,
+  MOCK_PLANS,
+  expectNockScopesDone,
+  defaultAppContextValue,
+  MockApp,
+  setupMockConfig,
+  mockConfig,
+  mockServerUrl,
+  mockOptionsResponses,
+} from '../../lib/test-utils';
+
+jest.mock('../../lib/sentry');
+
+jest.mock('../../lib/flow-event');
+
+import { SignInLayout } from '../../components/AppLayout';
+import Checkout from './index';
+import { AppContextType } from '../../lib/AppContext';
+import { PLANS } from '../../lib/mock-data';
+
+describe('routes/Checkout', () => {
+  let authServer = '';
+  let profileServer = '';
+
+  beforeEach(() => {
+    setupMockConfig({
+      ...mockConfig,
+      productRedirectURLs: PRODUCT_REDIRECT_URLS,
+    });
+    authServer = mockServerUrl('auth');
+    mockOptionsResponses(authServer);
+    profileServer = mockServerUrl('profile');
+    mockOptionsResponses(profileServer);
+  });
+
+  afterEach(() => {
+    noc.cleanAll();
+    return cleanup();
+  });
+
+  const Subject = ({
+    productId = PRODUCT_ID,
+    planId,
+    accountActivated,
```

Comment:
Is this query param used for anything? I got this from the `Product` route. It's passed into `SubscriptionCreate` there, but AFAICT, nothing is done with it...

Reply:
I'm not seeing a defined prop in `<SubscriptionCreate/>` for `accountActivated`. You are safe to remove.